docs/tutorials/local_iteration_series/local-Iteration-testing-multiplayer-games-locally.md

```diff
@@ -0,0 +1,62 @@
+
+## Player Builds
+
+First we need to build an executable. The default way of doing that is via  `File->Build Settings` in the menu bar, and then pressing `Build` button.
+
+Once the build has completed you can launch several instances of the built executable in order to both host and join a game.
+
+It is also possible to run the builds along with an editor that produced said build, which could be useful during iterations.
+
+> Mac users: to run multiple instances of the same app, you need to use the command line.
+> Run `open -n YourAppName.app`
+
+This approach, though functional, is somewhat inconvenient and slow - thus we list another option - ParrelSync.
+
+## ParrelSync
+![parrelsync-bossroom-demo](/static/img/parrelsync-bossroom-demo.gif)
+
+[ParrelSync](https://github.com/VeriorPies/ParrelSync) is an open-source Unity editor extension that allows users to **test multiplayer gameplay without building the project** by having another Unity editor window opened and mirror the changes from the original project. 
+
+ParrelSync works by making a copy of the original project folder and creating symbolic links to the `Asset` and `Project Settings` folders back from the original project.
+
+We use ParrelSync for local iteration in [BossRoom sample](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/).
+
+### Installation
+1.  Backup your project folder or use a version control system such as [Git](https://git-scm.com/), [SVN](https://subversion.apache.org/), [Plastic](https://www.plasticscm.com/) or any other.
+2.  Download .unitypackage from the [latest release](https://github.com/VeriorPies/ParrelSync/releases) and import it to your project.
+3.  ParrelSync should appear in the menu item bar after it has imported
+
+### Usage
+ - Open the `ParrelSync->Preferences` menu in the menu bar to open the preferences window
+ - Verify that your settings are set to the following: ![parrelsync-preferences](/static/img/parrelsync-preferences.png)
+:::important
+
+By default ParrelSync prevents asset serialization in all clone instances and changes can only be made from the original project editor. This is a **very important setting** that prevents issues with multiple editors accessing the same `Library` folder (which is not supported and breaks basic assumptions in Unity design).
+
+:::
+
+ - Open the `ParrelSync->Clones Manager` from which you can launch, create and remove clone editors.
+	 - Advanced usage is to utilize ParrelSync's capability of passing [Arguments](https://github.com/VeriorPies/ParrelSync/wiki/Argument) to clones, thus allowing to run custom logic on a per-clone basis.
+
+### Known issues and workarounds
+ - An important nuance is that ParrelSync does not sync changes made to packages. `Packages` folder is synced on clone opening, so if you made package changes - you should close and re-open your clones.
```

Comment:
If there's a github issue on their repo, let's link to it so users know the latest news from this. Our doc shouldn't refer to info that can easily be outdated without a way to get the latest info

Reply:
ParrelSync documentation is incorrect in multiple places - for instance, it states on the front page that `Packages` folder is symlinked, whereas it is not. Hence me pointing out known issues as of today.